workitem/workitemtype_repository.go

```diff
@@ -111,8 +112,9 @@ func ClearGlobalWorkItemTypeCache() {
 
 // Create creates a new work item in the repository
 // returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition) (*WorkItemType, error) {
+func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition, categories []*uuid.UUID) (*WorkItemType, error) {
```

Comment:
Do we have to add the category during WIT creation? Can't we just add it later? It would limit the places that we have to modify. Also, technically speaking, a WIT doesn't need a category to exist. Or did I miss a change to the `work_item_types` database table?